samples/net/wpanusb/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _wpanusb-sample:
+
+WPANUSB sample
+##############
+
+Overview
+********
+
+This application exports ieee802154 radio over USB to be used in other
+Operation Systems like Linux. In this scenario Linux SoftMAC driver would
+be used implementing ieee802154 stack inside Linux.
+
+Requirements
+************
+
+- Linux machine host connected via USB to Zephyr WPANUSB enabled MCU
```

Comment:
How about this:

    - a Zephyr WPANUSB enabled board (such as the :ref:`nrf52840_pca10056`)
      connected via USB to a Linux host.